tests/suites/test_suite_gcm.function

```diff
@@ -233,6 +233,90 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void gcm_crypt_and_tag_reuse_ctx(int cipher_id, data_t *key_str,
+                                 data_t *src_str, data_t *iv_str,
+                                 data_t *add_str, data_t *dst,
+                                 int tag_len_bits, data_t *tag,
+                                 int init_result)
+{
+    #define ITERATIONS 100
+    unsigned char output_enc[128];
+    unsigned char output_dec[128];
+    unsigned char tag_output[16];
+    unsigned char output_enc_rnd[128 * ITERATIONS];
+    unsigned char tag_output_rnd[16 * ITERATIONS];
+    mbedtls_gcm_context ctx;
+    size_t tag_len = tag_len_bits / 8;
+    int nloop;
+
+    mbedtls_gcm_init(&ctx);
+
+    memset(output_enc, 0x00, 128);
+    memset(output_dec, 0x00, 128);
+    memset(tag_output, 0x00, 16);
+
+    /* Allocate a buffer for random input data */
+    unsigned char rand_input[128];
+    memset(rand_input, 0x00, 128);
+
+    TEST_EQUAL(mbedtls_gcm_setkey(&ctx, cipher_id, key_str->x,
+                                  key_str->len * 8), init_result);
```

Comment:
This line is hard to read, with `init_result` appearing to be part of the parameters to the function call on the line above.

Might be better to either align `init_result` with `mbedtls_gcm_setkey` (if uncrustify allows that) or - while I really don't like "Yoda" style tests such as `if (0 != foo)` - do the test the other way round

```
    TEST_EQUAL(init_result, mbedtls_gcm_setkey(&ctx, cipher_id, key_str->x,
                                               key_str->len * 8));
```

Reply:
I just moved the `key_str->len * 8`  in one line and aligned `mbedtls_gcm_setkey` and `init_result`. The resulting line is 82 characters long, which is ok.